app/backend/src/couchers/servicers/discussions.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import grpc
+
+from couchers import errors
+from couchers.db import session_scope
+from couchers.models import Cluster, Discussion, Node, Thread
+from couchers.servicers.threads import pack_thread_id
+from couchers.utils import Timestamp_from_datetime
+from pb import discussions_pb2, discussions_pb2_grpc
+
+
+def discussion_to_pb(discussion: Discussion, user_id):
+    return discussions_pb2.Discussion(
+        discussion_id=discussion.id,
+        slug=discussion.slug,
+        created=Timestamp_from_datetime(discussion.created),
+        creator_user_id=discussion.creator_user_id,
+        owner_group_id=discussion.owner_cluster.id
+        if discussion.owner_cluster.official_cluster_for_node_id is None
+        else None,
```

Comment:
Ugly formatting, can you precalculate on a separate line or put parentheses around?

Reply:
Agree that it's ugly, I'll see if I can make black format it nicer with parentheses.

Reply:
Fixed with precomputation